.cirrus.yml

```diff
@@ -186,6 +192,31 @@ build_task:
     always: *runner_stats
 
 
+build_aarch64_task:
+    alias: 'build_aarch64'
+    name: 'Build for $DISTRO_NV'
+    # Multiarch doesn't depend on buildability in this automation context
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: "$CIRRUS_CRON != 'multiarch'"
+    ec2_instance: &standard_build_ec2_aarch64
+      image: ${FEDORA_AARCH64_AMI_ID}
+      type: t4g.micro
+      region: us-east-1
+      architecture: arm64
+    env:
+        <<: *stdenvars
+        DISTRO_NV: ${FEDORA_NAME_AARCH64}
+        TEST_FLAVOR: build
```

Comment:
For consistency, this should probably say `DISTRO_NV: ${FEDORA_NAME}` and `VM_IMAGE_NAME: ${FEDORA_NAME_AARCH64}`